### DISCUSSION TITLE
مشکل متصل نشدن به http://0.0.0.0:8000/

### DISCUSSION THREAD

**Discussion by 8MaCaVeL8** [Gozargah/Marzban#1434] [Q&A] [answered]
سلام، وقتتون بخیر دوستان
من بعد از اینکه مرزبان رو نصب کردم و رمز و ادمینش رو هم گذاشتم، داخل http://IP:8000/ که میرم عکس زیر رو نمایش میده و با انواع نسخه اوبنتو هم امتحان کردم در HostVDS ، لطفا راهنماییم کنید
![image](https://github.com/user-attachments/assets/7acc928b-1ad1-416a-9f4d-29e96ee36694)

**Comment by AmirKenzo** [comment]
سلام برای پنل باید ssl بگیرید https://github.com/Gozargah/Marzban/discussions/188

**Comment by ByteMysticRogue** [comment, marked as answer]
سلام، اگر میخواید که پنل رو لوکال باشه میتونید ssh -L 8000:localhost:8000 username@ip_server یا ریورس پراکسی بکنید با nginx بعد ازینکه ssl تهیه کردید. یا میتونید ssl تهیه کنید و تو فایل .env مسیرش قرار بدین.
